# Description: Install docs to /usr/share/man instead of /usr/man.

--- guilt-0.36/Documentation/Makefile.~2~	2018-01-25 08:03:51.473420514 +0000
+++ guilt-0.36/Documentation/Makefile	2018-01-25 08:03:51.504530220 +0000
@@ -12,7 +12,7 @@
 prefix?=$(PREFIX)
 bindir?=$(prefix)/bin
 htmldir?=$(prefix)/html
-mandir?=$(prefix)/man
+mandir?=$(prefix)/share/man
 man1dir=$(mandir)/man1
 man7dir=$(mandir)/man7
 # DESTDIR=
@@ -34,7 +34,7 @@
 man1: $(DOC_MAN1)
 man7: $(DOC_MAN7)
 
-install: man
+install:
 	$(INSTALL) -d -m755 $(DESTDIR)$(man1dir) $(DESTDIR)$(man7dir)
 	$(INSTALL) -m644 $(DOC_MAN1) $(DESTDIR)$(man1dir)
 	$(INSTALL) -m644 $(DOC_MAN7) $(DESTDIR)$(man7dir)
